metaworld/envs/mujoco/sawyer_xyz/base.py

```diff
@@ -110,6 +131,7 @@ def set_xyz_action(self, action):
         self.data.set_mocap_quat('mocap', np.array([1, 0, 1, 0]))
 
     def discretize_goal_space(self, goals):
+        assert False
```

Comment:
Obviously we should either delete these methods or remove the asserts.

Reply:
perhaps we should raise a RuntimeError and point the user to instructions for the new API?